src/v2/manifest/manifest_schema2.rs

```diff
@@ -19,6 +26,12 @@ pub struct Config {
     digest: String,
 }
 
+/// Partial representation of a container image (application/vnd.docker.container.image.v1+json)
```

Comment:
Is the full version of this described somewhere?

Reply:
I didn't find it fast enough when I was implementing it, so I just picked the field which I was interested at the time.

Reply:
I want to add that this was meant as an RFC, and I'll gladly implement the whole structure instead of just the field, and also reference the spec for it ;-)

Reply:
I think this is described here https://github.com/moby/moby/blob/master/image/spec/v1.md#image-json-description.

Reply:
Would you  prefer to have the full struct implemented for completeness?

Reply:
I think that's overkill in this scope. Just link to the spec and leave a note that there are remaining fields that could be added in the future.

Reply:
Thanks for digging for the spec. It's referenced int the docstring now.